15/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor-tinymce/plugins.md

```diff
@@ -2,19 +2,17 @@
 description: Information on how to work with TinyMCE plugins in the rich text editor.
 ---
 
-# Rich Text Editor Plugins
-
-## Overview
+# Plugins
 
 The Rich Text Editor (RTE) in Umbraco is based on the open source editor [TinyMCE](https://www.tiny.cloud/). TinyMCE is a highly customizable editor, and it is possible to extend the functionality of the editor by adding plugins.
 
 TinyMCE comes with a lot of plugins out of the box, but it is also possible to create your own plugins. This article will show you how to add a custom plugin to the rich text editor.
 
-### Open-Source Plugins
+## Open-Source Plugins
 
 TinyMCE has a lot of open-source plugins available. You can find a list of these plugins on the [TinyMCE website](https://www.tiny.cloud/docs/tinymce/6/plugins/#open-source-plugins).
 
-### Premium Plugins
+## Premium Plugins
 
 TinyMCE also has a number of [premium plugins](https://www.tiny.cloud/docs/tinymce/6/plugins/#premium-plugins) available. These plugins are only available for [paid TinyMCE subscriptions](https://www.tiny.cloud/pricing/). They can be added to the rich text editor by [adding a bit of configuration](#adding-a-premium-plugin).
 
```

Comment:
I know you didn't change the text here but noticed these lists need space after the number 😀 
1.Add a manifest file
2.Add the plugin.js file